### PR TITLE
Update EIP-5521: Add more descriptions

### PR DESCRIPTION
**Pull request by OniReimu**
1. Add a description of the collaboration with the eip-998
2. Add customized attribute data fields for specific scenarios
3. Remove 5006 from `required` as 5006 is merely a protocol to collaborate with rather than a compulsory basis, similar to the newly-added 998.

### CODE REVIEW

**Review by xinbenlv — COMMENTED** [review]
@OniReimu thanks for the PR
First of all, is it creating a new EIP or is a updating descriptions?

**Review by Pandapip1 — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts

**Comment by OniReimu** [comment]
> 

Hello @xinbenlv , this is essentially an update for EIP-5521, not a new eip proposal.

**Comment by OniReimu** [comment]
> 

Conflicts fixed, thank you @ Pandapip1 :)

File: EIPS/eip-5521.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
 eip: 5521
 title: Referable NFT
 description: An EIP-721 extension to construct reference relationships among NFTs
-author: Saber Yu (@OniReimu), Qin Wang <qin.wang@data61.csiro.au>, Shange Fu <shange.fu@monash.edu>, Shiping Chen <shiping.chen@data61.csiro.au>, Sherry Xu <xiwei.xu@data61.csiro.au>, Jiangshan Yu <jiangshan.yu@monash.edu>
+author: Saber Yu (@OniReimu), Qin Wang <qin.wang@data61.csiro.au>, Shange Fu <shange.fu@monash.edu>, Shiping Chen <shiping.chen@data61.csiso.au>, Sherry Xu <xiwei.xu@data61.csiro.au>, Jiangshan Yu <jiangshan.yu@monash.edu>
```
Comment (Pandapip1) [inline EIPS/eip-5521.md]: Is this change intentional?

File: .gitignore [hunk 2]
```diff
@@ -9,3 +9,4 @@ vendor
 
 # Secrets
 .vercel
+.DS_Store
```
Comment (Pandapip1) [inline .gitignore:12]: Please put this in a separate PR.

File: EIPS/eip-5521.md (hunk 1 above)
Comment (OniReimu) [reply EIPS/eip-5521.md]: `csiso` is a typo.

File: .gitignore (hunk 2 above)
Comment (OniReimu) [reply .gitignore]: Remove this change

**Comment by Ellajoke** [comment]
Approve

**Review by Pandapip1 — APPROVED** [review]

**Comment by Pandapip1** [comment]
(FYI for any concerned editors: @eth-bot was passing, so I was able to merge it)

**Pull request merged**